packages/advisor-components/package.json

```diff
@@ -29,8 +29,8 @@
         "@patternfly/react-core": ">=4.162.16",
         "@patternfly/react-icons": ">=4.3.5",
         "prop-types": ">=15.6.2",
-        "react": ">=16.14.0 || >=17.0.0",
-        "react-dom": ">=16.14.0 || >=17.0.0",
+        "react": "^16.14.0",
```

Comment:
Is there any reason you do not include the v17 here? If it's unable to parse the version use 17 instead. Even that is no longer lts.

Reply:
@Hyperkid123, TBH, I don't remember the exact problem: I couldn't install all the required dependencies. However, now I tried to revert this change, then run `npm run clean && npm i`, and it looks like it builds fine. So I changed the versions back at https://github.com/RedHatInsights/frontend-components/pull/1540/commits/85aa3b3b64ddde6a6b17bb8da16fdc2464eef308